src/Compiler/Driver/fsc.fs

```diff
@@ -671,6 +671,9 @@ let main1
     // Build the initial type checking environment
     ReportTime tcConfig "Typecheck"
 
+    // Read the source file content for the `CallerArgumentExpression` feature
+    readAndStoreFileContents tcConfig sourceFiles
```

Comment:
Could you also add a "ReportTime" section here so that we can see impact on timing in selected compilations?

Reply:
Especially something bigger, like when the compiler compiles itself (this will need the `--times` flag to report the timings)